src/response/topology/topologyNodeResponse.ts

```diff
@@ -15,6 +15,7 @@ export interface TopologyNode {
   workflowId?: string;
   localRun?: boolean;
   url?: string;
+  activeThresholds?: ActiveThresholdsItem;
```

Comment:
Why is activeThresholds plural while having a single icon item?
Is its usage "if the node has `activeThresholds` show this icon"?

Reply:
As discussed, will be multiple in the future